### ISSUE TITLE
IDE support

### ISSUE THREAD

**Issue by arsenij-melihov** [allegro/typescript-strict-plugin#20]
Hi!
Is there a way to get errors in IDE? 
[README](https://github.com/allegro/typescript-strict-plugin#do-i-need-this-plugin) says "They will show errors in your IDE but they won't appear during compilation", but I didn't find a way to get them. 
(I am using vscode).

Thanks in advance!

**Comment by KostkaBrukowa** [comment]
Can you share a link to your repository?

**Comment by arsenij-melihov** [comment]
I created a reproduction repo - https://github.com/arsenij-melihov/tsc-strict-ide-example. 
It contains `./src/index.ts` file with the following code: 
```ts
//@ts-strict
interface TestType {
  bar: string;
}

const foo: TestType | undefined = undefined;

console.log(foo.bar);
```
and default `typescript-strict-plugin` setup. 

And vscode doesn't show me errors: 
![image](https://user-images.githubusercontent.com/88650395/150518642-5e37bc30-eaac-4d49-83b2-fbd0d995b764.png)

**Comment by KostkaBrukowa** [comment]
I believe it is the same problem as with https://github.com/allegro/typescript-strict-plugin/issues/5#issuecomment-882106427.
You need to choose your tsconfig as a main typescript configuration file and it should work

**Comment by arsenij-melihov** [comment]
You are absolutely, right!
